src/google/protobuf/descriptor.proto

```diff
@@ -311,8 +311,8 @@ message MethodDescriptorProto {
 //   number by putting them in a sub-message. See the Custom Options section of
 //   the docs for examples:
 //   https://developers.google.com/protocol-buffers/docs/proto#options
-//   If this turns out to be popular, a web service will be set up
-//   to automatically assign option numbers.
+//   You may also send a pull request or file an issue: see
```

Comment:
Thanks for the cleanup. Can you change the text above "e-mail protobuf-global-extension-registry@..." to:

To obtain globally unique field numbers, please send a request to add an entry to [protobuf global extension registry](https://github.com/protocolbuffers/protobuf/blob/master/docs/options.md). 

? We no longer use the email group.

Reply:
ptal